### PR TITLE
order items by name

### PR DESCRIPTION
**Pull request by maryamaltawil4**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -124,6 +130,57 @@ for (var i = 0 ; i < courses.length ; i++) {
...
+        if (a.name > b.name) {
+          return 1;
+        }
+        return 0;
+      });
+      
+      console.log(courses); 
+      display();
+      sortName.style.display="none";
+      sortNameDown.style.display="inline-block";
```
Comment (IbrahimLama) [inline main.js]: I see sortName.onclick and sortNameDown.onclick, are almost the same functionality but with an oposite behaviour, can you refactor the code to have only one function and reuse it based if you want to sort up or down

Comment (maryamaltawil4) [reply main.js]: the code update by one function

File: main.js [hunk 2]
```diff
@@ -124,8 +130,38 @@ for (var i = 0 ; i < courses.length ; i++) {
...
+}
+// to display course without sort 
+withoutSort.onclick = function(){  
+    courses=sortElement ;
+     display();
+     sortName.style.display="inline-block";
+     withoutSort.style.display= "none";
+}
+// to display function toggleSortOrder when we click at sort/sortdowm 
+sortName.onclick = toggleSortOrder;
```
Comment (IbrahimLama) [inline main.js]: can you add the onClick event from the html attribute instead of attaching it here ?

File: main.js [hunk 3]
```diff
@@ -124,8 +130,36 @@ for (var i = 0 ; i < courses.length ; i++) {
    }
 }
 
+sortNameDown.style.display="none";
+withoutSort.style.display="none";
+sortName.style.display= "inline-block" ;
+
+function toggleSortOrder() {
```
Comment (IbrahimLama) [inline main.js:137]: we can pass to this function the type of sort (up or down) and based on it we can do the switch statement

**Pull request closed without merging**